lib/chef/knife/winrm_shared_options.rb

```diff
@@ -27,16 +27,16 @@ module WinrmSharedOptions
       # Shared command line options for knife winrm and knife wsman test
       def self.included(includer)
         includer.class_eval do
-          option :manual,
+          option :manual_list,
             :short => "-m",
             :long => "--manual-list",
             :boolean => true,
             :description => "QUERY is a space separated list of servers",
             :default => false
 
-          option :attribute,
-            :short => "-a ATTR",
-            :long => "--attribute ATTR",
+          option :winrm_attribute,
```

Comment:
Ah but you don't, especially in light of my changes to lib/chef/knife/knife_windows_base.rb #locate_config_value and the fact that ssh.rb's `:attribute` resolves to `:ssh_attribute`. See: #290 

Namespacing these attributes (both here and in core ssh) allows the code to be more explicit and resolves the stated conflict with `knife node show`.